tools/run_tests/helper_scripts/build_python.sh

```diff
@@ -122,7 +122,7 @@ export LANG=en_US.UTF-8
 
 # Allow build_ext to build C/C++ files in parallel
 # by enabling a monkeypatch. It speeds up the build a lot.
-export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=4
+export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=${GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS:-4}
```

Comment:
:100: Very nice change!

Can we try setting this to something more intelligent by default? I believe the system is assumed to be posixy for this shell script. Maybe `grep ^cpu\\scores /proc/cpuinfo | uniq |  awk '{print $4}'`?

Reply:
Sadly, macOS doesn't have `/proc/cpuinfo`.

Reply:
For me, I added `export GRPC_PYTHON_BUILD_EXT_COMPILER_JOBS=12` to my `.bashrc`.

Reply:
12 CPUs? lucky!  I would use that trick.

Reply:
Added `DEFAULT_PARALLEL_JOBS=`nproc` || DEFAULT_PARALLEL_JOBS=4` to automate the speed up.